torch/_refs/linalg/__init__.py

```diff
@@ -101,7 +102,7 @@ def diagonal(
 @out_wrapper(exact_dtype=True)
 def vector_norm(
     x: TensorLikeType,
-    ord: float = 2.0,
+    ord: Union[float, int] = 2.0,
```

Comment:
bah, maybe we should default this to 2 (int) lol

Reply:
Thanks, addressed